launcher/modplatform/flame/FlameInstanceCreationTask.cpp

```diff
@@ -259,6 +255,57 @@ bool FlameCreationTask::updateInstance()
     return false;
 }
 
+QString FlameCreationTask::getVersionForLoader(QString uid, QString loaderType, QString loaderVersion, QString mcVersion)
+{
+    if (loaderVersion == "recommended") {
+        auto vlist = APPLICATION->metadataIndex()->get(uid);
+        if (!vlist) {
+            setError(tr("Failed to get local metadata index for %1").arg(uid));
+            return {};
+        }
```

Comment:
Has no effect

Reply:
I'm sure it does in InstanceCreationTask that error is checked

Reply:
vlist will never be null

Reply:
well then that sucks:  I see this used in a similar way on multiple places: VersionPage and ATLPackInstall.
All of them have the same approach I use here.
So I would prefer to keep it as is in order to avoid any crashes.

Reply:
ah, fair enough